OData/src/System.Web.OData/OData/Query/OrderByOpenPropertyNode.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Web.Http;
+using System.Web.OData.Properties;
+using Microsoft.OData.Core;
+using Microsoft.OData.Core.UriParser.Semantic;
+
+namespace System.Web.OData.Query
+{
+    /// <summary>
+    /// Represents ordering on a dynamic property
+    /// </summary>
+    public class OrderByOpenPropertyNode : OrderByNode
+    {
+        /// <summary>
+        /// Default constructor for a dynamic property order by
```

Comment:
I will change the comment though it's hardly informing that a constructor initializes a new instance